postgres/defaults.yaml

```diff
@@ -39,3 +39,30 @@ postgres:
     command: service postgresql initdb
     test: test -f /var/lib/pgsql/data/PG_VERSION
     env: {}
+
+  # Client executables list
+  client_bins:
+    - clusterdb
+    - createdb
+    - createlang
+    - createuser
+    - dropdb
+    - droplang
+    - dropuser
+    - pg_archivecleanup
+    - pg_basebackup
+    - pg_config
+    - pg_dump
+    - pg_dumpall
+    - pg_isready
+    - pg_receivexlog
+    - pg_restore
+    - pg_rewind
+    - pg_test_fsync
+    - pg_test_timing
+    - pg_upgrade
+    - pg_xlogdump
```

Comment:
@javierbertoli Thanks for review. Indeed, because this addition is totally RedHat specific (for upstream packages), its definitely better to set `client_bins` in the `osmap.yaml`. I'll do that.

But, the `bin_dir` item already belongs to the `RedHat` section in the `osmap` after the "`{% if repo.use_upstream_repo %}`" switch. Also, I've added `onlyif` requisite to the `alternatives.install` state in the `client.sls`. So, the cases with other platforms and even some missing binaries are covered as well.